stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_reply_view.xml

```diff
@@ -61,7 +61,6 @@
             android:id="@+id/replyText"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:autoLink="web"
```

Comment:
We also need to remove this from `TextView` used in old component module, right?

Reply:
Done.